mars/core/entrypoints.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import warnings
+import functools
+
+from pkg_resources import iter_entry_points
+
+logger = logging.getLogger(__name__)
+
+
+# from https://github.com/numba/numba/blob/master/numba/core/entrypoints.py
+# Must put this here to avoid extensions re-triggering initialization
+@functools.lru_cache(maxsize=None)
+def init_all():
```

Comment:
Use a more meaningful method name, for instance, `init_extension_entrypints`?

Reply:
Yes, `init_extension_entrypoints` is better. Thanks.